docs/content/design/registry.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Registry"
```

Comment:
This is `registry.md` and titled registry but details the proxy.

Reply:
Wild copy paste daemon appeared